### ISSUE TITLE
Password protected shares do not honor config.json settings

### ISSUE THREAD

**Issue by jimeelicious** [alangrainger/immich-public-proxy#9]
When Immich shares have no password, `config.json` settings seem to be honored.
![image](https://github.com/user-attachments/assets/38415a4f-4dec-4c1f-999b-e03b664ccbde)

`# cat config.json `
```
{
  "lightGallery": {
    "controls": true,
    "download": true,
    "mobileSettings": {
      "controls": false,
      "showCloseIcon": true,
      "download": true
    }
  }
}
```
Mobile view:
![image](https://github.com/user-attachments/assets/c94331c2-81df-45dc-bda3-12adcf82be90)

But when setting a password in the Immich share, and using the generated sharing link 
![image](https://github.com/user-attachments/assets/dbb0dadb-12d5-4f4c-93a0-3f69748912d8)

The settings do not appear to be honored, such as missing the download and close icons from mobile view:
![image](https://github.com/user-attachments/assets/3482d841-7552-437a-8e61-e54a0a98ba32)

I apologize that I do not have enough experience to look deeply into the root cause in the code, but I suspect potentially `password.ejs` is not somehow drawing this configuration in when loading.

A temporary fix would be to modify `public/lightgallery.min.js` code attributes directly for the `showCloseIcon` and `download` properties within `mobileSettings`, but this is not ideal as it modifies the lightGallery base code
```
...  mobileSettings:{controls:!1,showCloseIcon:!1,download:!1}  ...
```

**Comment by alangrainger** [comment]
Thanks! - this is now fixed in v1.3.3

**Comment by jimeelicious** [comment]
Wow thanks for the fast reply.

I was only able to get as far as adding `initLightGallery(<%- JSON.stringify(lgConfig) %>) // initLightGallery imported from web.js` in `views/password.ejs` on my own.

Reading your changes, it seems `src/immich.ts` needed to be updated too. Thanks again!
